lib/logger.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+let logger = null;
+
+const set = l => logger = l;
+const get = () => logger;
+
+const isDebug = () => logger && logger.debug;
```

Comment:
I find these names misleading, something like `hasDebug` etc would be better, IMO. I'd expect `isDebug`, `isVerbose` and alike to report the current logging level, which is not the case with these checks because these methods will be present in virtually every logger regardless of the logging level.